dcrec/edwards/curve_test.go

```diff
@@ -65,9 +65,8 @@ func TestXRecovery(t *testing.T) {
 				t.Fatalf("expected %v, got %v", false, notOnCurve)
 			}
 			b2 := BigIntPointToEncodedBytes(x2, y)
-			cmp := bytes.Compare(vector.bIn[:], b2[:]) == 0
-			if !cmp {
-				t.Fatalf("expected %v, got %v", true, cmp)
+			if !bytes.Equal(vector.bIn[:], b2[:]) {
+				t.Fatalf("expected %v, got %v", vector.bIn[:], b2)
```

Comment:
I would suggest using `%x` instead of `%v` since there are byte slices.  The display is much more useful in hex.

Reply:
changes made